doc/environment.yml

```diff
@@ -14,5 +14,6 @@ dependencies:
   - pandoc
   - pip
   - pip:
+    - doc8
```

Comment:
Out of curiosity, is there a reason to install this with `pip` instead of `conda`?

Reply:
Nope - changed

Reply:
Actually yes - it seems the read the docs build can't find doc8 with conda. Works with pip so changing back for now

Reply:
Ah, I see. I think that's because `doc8` is only on `conda-forge` and not the `defaults` channel. I'm proposed we switch to `conda-forge` over in https://github.com/xgcm/xhistogram/pull/39, so maybe we should do the same for `doc/environment.yml ` here?

Reply:
No a big deal either way, I don't mean for this comment to be a blocker in any way